CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
   [#553](https://github.com/OpenEnergyPlatform/open-MaStR/issues/553)
 - Allow to configure model/service port in `soap_api.download.MaStRAPI`
   [#556](https://github.com/OpenEnergyPlatform/open-MaStR/issues/556)
+- Allow CSV export of table `storage_units`
+  [#562](https://github.com/OpenEnergyPlatform/open-MaStR/issues/562)
```

Comment:
We used the pull request (/pull/565) instead of the issue here. I don't know why, but maybe because of the auto generated Changelog during the release procedure.

Reply:
Done. The other entries should be changed then as well, but let's do that on dev later.